web/components/contract/quick-bet.tsx

```diff
@@ -8,31 +12,31 @@ import {
   DPM,
   Binary,
   NumericContract,
-  FreeResponse,
+  FreeResponseContract,
 } from 'common/contract'
-import { formatMoney } from 'common/util/format'
+import {
+  formatLargeNumber,
+  formatMoney,
+  formatPercent,
+} from 'common/util/format'
+import { useState } from 'react'
 import toast from 'react-hot-toast'
 import { useUser } from 'web/hooks/use-user'
 import { useUserContractBets } from 'web/hooks/use-user-bets'
 import { placeBet } from 'web/lib/firebase/api-call'
-import { getBinaryProb } from 'web/lib/firebase/contracts'
+import { getBinaryProb, getBinaryProbPercent } from 'web/lib/firebase/contracts'
 import TriangleDownFillIcon from 'web/lib/icons/triangle-down-fill-icon'
 import TriangleFillIcon from 'web/lib/icons/triangle-fill-icon'
 import { Col } from '../layout/col'
 import { OUTCOME_TO_COLOR } from '../outcome-label'
 import { useSaveShares } from '../use-save-shares'
-import {
-  BinaryResolutionOrChance,
-  NumericResolutionOrExpectation,
-  FreeResponseResolutionOrChance,
-} from './contract-card'
 
 export function QuickBet(props: { contract: Contract }) {
   const { contract } = props
 
   const user = useUser()
   const userBets = useUserContractBets(user?.id, contract.id)
-  const { yesFloorShares, noFloorShares, yesShares, noShares } = useSaveShares(
+  const { yesFloorShares, noFloorShares } = useSaveShares(
```

Comment:
And actually, as it is written it's not accurate b/c if I have bets on any outcome it'll show that I have yes shares

Reply:
Yeah, good point on the bug, and thanks for the explanation. I think fixing this is a bit annoying since useSaveShares is utilized in a bunch of different places; a visual indicator of "I did bet on this market" is okay for now.

Since this wasn't a bug introduced by this PR (it's already in `main`), I'm going to continue with shipping this.